sphinx/source/docs/user_guide/server.rst

```diff
@@ -101,8 +101,63 @@ Streaming Data
 
 Large Data
 ----------
+This feature is currenlty in beta.  Bokeh supports large datasets via the blaze, combined with a few abstract rendering and smart downsampling routines.  Bokeh currently pulls in the blaze server as a flask blueprint - so if you have the necessary libraries installed, your bokeh server is also a blaze server.
 
+Blaze Server
+------------
+
+To use the smart downsampling facilities of bokeh, you must install the multi-user blaze server, https://github.com/ContinuumIO/multiuserblazeserver.::
+
+    $ conda install -c bokeh multiuserblazeserver
+
+or::
+
+    $ pip install -c bokeh multiuserblazeserver
```

Comment:
We should coordinate with mrocklin so there is just `blaze-server` with a multi-user mode